src/display/lv_display.c

```diff
@@ -924,6 +924,39 @@ lv_draw_buf_t * lv_display_get_buf_active(lv_display_t * disp)
     return disp->buf_act;
 }
 
+void lv_display_rotate_area(lv_display_t * disp, lv_area_t * area)
+{
+    lv_display_rotation_t rotation = lv_display_get_rotation(disp);
+
+    if(rotation == LV_DISPLAY_ROTATION_0) {
+        return;
+    }
+
+    int32_t w = lv_area_get_width(area);
+    int32_t h = lv_area_get_height(area);
+
+    if(rotation == LV_DISPLAY_ROTATION_90) {
+        area->y2 = disp->ver_res - area->x1 - 1;
```

Comment:
I thought you would set the properties in this order (which is more understandable): `x1`, `y1`, `x2`, `y2`

Reply:
It's somewhat complicated because the last two assignments to `x2` and `y1` reuse the first two assignments to `x1` and `y2`. I suppose we could expand this at the cost of being slightly more verbose but I don't feel very strongly either way.

Reply:
I think it'sokay as it is. Not the most elegant, but IMO adding verbose comments would be overkill too.

Reply:
> It's somewhat complicated because the last two assignments to `x2` and `y1` reuse the first two assignments to `x1` and `y2`. I suppose we could expand this at the cost of being slightly more verbose but I don't feel very strongly either way.

Oh, you are totally right, I forget that order is here important. Mea culpa.
No need for comment.